doc/api/console.markdown

```diff
@@ -32,7 +32,7 @@ In daily use, the blocking/non-blocking dichotomy is not something you
 should worry about unless you log huge amounts of data.
 
 
-### console.log([data][, ...])
+### console.log(object [, object, ...])
```

Comment:
The first argument is optional too. You can call `console.log()` to print a blank line.